src/health_misinfo_shared/fine_tuning.py

```diff
@@ -278,7 +360,12 @@ def get_video_responses(
         )
 
     for chunk in chunks:
-        prompt = f"{infer_prompt}\n```{chunk['text']}```"
+        # chunk_text = chunk["text"] if "text" in chunk.keys() else chunk["input_text"]
+        try:
```

Comment:
This logic suggests we don't know what datatype chunk will be. I think it used to be a string but is now a dict containing `text`. It should certainly always be the same type, so this logic is likely to hide any errors elsewhere.